website/management/commands/leaderboard.py

```
@@ -1,27 +1,43 @@
-from django.contrib.auth.models import User
+from django.db import transaction
 
-from website.management.base import LoggedBaseCommand
-from website.models import Issue, UserProfile
 
+@receiver(post_save, sender=DailyStatusReport)
```

Ruff (0.14.7)

Undefined name `receiver`

(F821)